$OpenBSD: patch-tests_common-pg_cpp,v 1.1.1.1 2015/10/30 17:18:53 landry Exp $

Fix compiler error:
	conversion from 'long int' to non-scalar type 'pg::result_ptr
	{aka boost::shared_ptr<pg::result>}' requested
	      
--- tests/common-pg.cpp.orig	Fri Aug 14 22:28:11 2015
+++ tests/common-pg.cpp	Fri Sep 18 23:10:29 2015
@@ -75,7 +75,7 @@ result::~result() {
 
 tempdb::tempdb()
     : m_conn(conn::connect("dbname=postgres")) {
-    result_ptr res = NULL;
+    result_ptr res;
     m_db_name = (boost::format("osm2pgsql-test-%1%-%2%") % getpid() % time(NULL)).str();
     m_conn->exec(boost::format("DROP DATABASE IF EXISTS \"%1%\"") % m_db_name);
     //tests can be run concurrently which means that this query can collide with other similar ones
